app/src/main/java/com/quran/labs/androidquran/view/AudioStatusBar.java

```diff
@@ -409,6 +453,83 @@ private void addDownloadProgress(@StringRes int text) {
     addView(ll, lp);
   }
 
+  @SuppressLint("NewApi")
```

Comment:
is this for the image tint stuff? can we remove this suppress lint and make sure that either the model has the correct minSdk version or do an sdk check instead? otherwise can we use something from appcompat?

Reply:
This `SupressLint` is not required. Removed.
(image tint is API 21+)